scripts/jenkins/groovy/compareBenchmarksStage.groovy

```diff
@@ -192,8 +192,8 @@ def call(final pipelineContext, final stageConfig, final benchmarkFolderConfig)
             ],
             'cox2': [
                 [10, "cpu"]: [
-                    'train_time_min': 1275,
-                    'train_time_max': 1464
+                    'train_time_min': 62,
```

Comment:
Do we really need such a difference between min and max value? Or do we understand why it's needed?

Reply:
The original time difference is in ms.  You have a good point, let me go back and check my results and update the max value as well.